src/components/Shop/Shop.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const Shop = () => {
+  return (
+    <div style={{ backgroundImage: "url(./shopimage.png)" }}>
```

Comment:
This should be done using tailwindcss
Checkout:
- https://tailwindcss.com/docs/background-image
- https://tailwindcss.com/docs/background-opacity
modify tailwind.config with the original image, and play with the opacity to match the one in Figma 
![images (1)](https://user-images.githubusercontent.com/69876518/127709561-e0ef60bc-6a5c-4758-9593-e2bd82fe2199.jpg)

Reply:
actually when I was testing it in the code its the same pic as the one in figma file .

Reply:
Try to do it using tailwind, same logic just in tailwind.config instead of style attribute:

- https://tailwindcss.com/docs/background-image
- https://tailwindcss.com/docs/background-opacity